src/javascript/app_2/App/Components/Form/DropDown/dropdown.jsx

```diff
@@ -34,14 +30,33 @@ class Dropdown extends React.Component {
         document.removeEventListener('mousedown', this.handleClickOutside);
     }
 
-    handleSelect(item) {
+    isSingleOption = isArrayLike(this.props.list) ?
```

Comment:
boolean should be underscore i think? `is_single_option`

Reply:
good spot, missed this 👍